axlearn/cloud/gcp/jobs/gke_runner.py

```diff
@@ -446,9 +446,18 @@ def from_flags(cls, fv: flags.FlagValues, **kwargs):
         return cfg
 
 
+class GPUGKERunnerJob(GKERunnerJob):
+    """A GKERunnerJob that uses GPUGKEJob."""
+
+    inner = GPUGKEJob
+    pre_provisioner = TPUNodePoolProvisioner
```

Comment:
A reminder to change `TPUNodePoolProvisioner` when ready. Maybe we can just use the default implementation that raises NotImplementedError.

Reply:
I somehow missed this! Thanks for catching it. Will fix

Reply:
fixed